src/pages/Search/EmptySearchView.tsx

```diff
@@ -272,7 +272,7 @@ function EmptySearchView({type, hasResults}: EmptySearchViewProps) {
                     titleStyles={content.titleStyles}
                     subtitle={content.subtitle}
                     buttons={content.buttons}
-                    headerContentStyles={[styles.h100, styles.w100, ...content.headerContentStyles]}
+                    headerContentStyles={[styles.h100, styles.w100, ...content.headerContentStyles] as Array<ViewStyle & ImageStyle>}
```

Comment:
Please revert this change , I don't think it's needed here.

Reply:
I need to either update the type of `headerContentStyles` props or keep it as is. Removing it results in a type error.

Reply:
If the styling is consistent across other `headerContentStyles`, why is `CONST.SEARCH.DATA_TYPES.TRIP` the only one throwing a type error?

Reply:
Other cases in the switch have `!hasResults`, which prevents the typescript from inferring their types well. for exp. 

If you return this for the first case, it leads to different type errors, which work correctly in the default case. 

```tsx 

  return {
                    headerMedia: LottieAnimations.GenericEmptyState,
                    title: translate('search.searchResults.emptyResults.title'),
                    subtitle: translate('search.searchResults.emptyResults.subtitle'),
                    headerContentStyles: [styles.emptyStateFolderWebStyles, StyleUtils.getBackgroundColorStyle(theme.emptyFolderBG)],
                    lottieWebViewStyles: {backgroundColor: theme.emptyFolderBG, ...styles.emptyStateFolderWebStyles},
                };

```